spot-price-monitor.py

```diff
@@ -71,6 +106,10 @@ def get_args():
     parser.add_argument('-r', '--region', type=str, default='us-east-1',
                         help='''The region that the cluster is running
                         in (Default: us-east-1)''')
+    parser.add_argument('-o', '--ondemand', action="store_true", default=False,
```

Comment:
Somehow I feel like `-o` should be for some kind of output since that what it's used for in kubernetes everywhere else.  I think it might be OK to have the long-form option only?

Reply:
I'll happily remove the -o if thats what it takes to get it accepted

Reply:
I've removed the -o short-form as discussed.